### PR TITLE
Bump all dependencies to latest

### PR DESCRIPTION
**Pull request by nyurik**
Maintenance PR to see if upgrading ALL dependencies to latest would trigger any CI errors. Feel free to create other PRs that upgrade only some of the dependencies. This does not have to be merged.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,114 +10,114 @@
...
   "engines": {
-    "node": ">=6.4.0"
+    "node": ">=12.13.0"
   },
   "dependencies": {
     "@mapbox/geojson-rewind": "^0.5.0",
     "@mapbox/geojson-types": "^1.0.2",
     "@mapbox/jsonlint-lines-primitives": "^2.0.2",
-    "@mapbox/mapbox-gl-supported": "^1.5.0",
+    "@mapbox/mapbox-gl-supported": "^2.0.0",
```
Comment (lseelenbinder) [inline package.json:19]: I'm guessing we don't actually want this to be v2?

Comment (nyurik) [reply package.json:19]: this one is tricky -- that lib is not about gl 2.0, but just a bump to the versions to keep them in sync. It doesn't even peer-depend on the GL lib at all... No idea, will need to investigate.

**Comment by HarelM** [comment]
Is this covered by #190?
Seems a bit more effort to migrate to node 12 than what's here, right?

**Comment by HarelM** [comment]
Closing this in favor of #190. If this is still relevant please open a new PR.

**Pull request closed without merging**